cluster/images/kubemark/Makefile

```diff
@@ -15,18 +15,17 @@
 # build Kubemark image from currently built binaries containing both 'real' master and Hollow Node.
 # This makefile assumes that the kubemark binary is present in this directory.
 
-REGISTRY?=gcr.io
-PROJECT?=google_containers
+REGISTRY?=k8s.gcr.io
 
 all: gcloudpush
 
 build:
-	docker build --pull -t $(REGISTRY)/$(PROJECT)/kubemark .
+	docker build --pull -t $(REGISTRY)/kubemark .
 
 gcloudpush: build
-	gcloud docker -- push $(REGISTRY)/$(PROJECT)/kubemark
+	gcloud docker -- push $(REGISTRY)/kubemark
```

Comment:
For all Makefiles performing pushes to k8s.gcr.io:
sed 's/gcloud docker -- push/gcloud docker --server=k8s.gcr.io -- push/g'
This'll be needed until k8s.gcr.io becomes a default registry for `gcloud docker`

Reply:
uggh.  Done.